src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+@import "../utils/_mixin.scss";
+@import "../utils/_placeholders.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.